File: Python/Iniciante/1020 - Idade em Dias.py
"""
Autor: Pierre Vieira
Data da submissão: 22/01/2019 23:55:21
"""
n_dias = int(input())
ano = n_dias//365
mes = (n_dias%365)//30
dias = (n_dias%365)%30
print("{} ano(s)\n{} mes(es)\n{} dia(s)".format(ano, mes, dias))

File: C#/Iniciante/1020 - Idade em Dias.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 14:24:54
*/
using System;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int dias, meses, anos;
            dias = int.Parse(Console.ReadLine());
            anos = dias / 365;
            dias %= 365;
            meses = dias / 30;
            dias %= 30;
            Console.WriteLine($"{anos} ano(s)\n{meses} mes(es)\n{dias} dia(s)");
        }
    }
}
